src/Components/PoiInformations.js

```diff
@@ -4,49 +4,60 @@ import { connect } from 'react-redux';
 
 const mapStateToProps = state => ({
   specificPoiInfos: state.specificPoiInfos,
+  InformationPoiInfos: state.InformationPoiInfos,
 });
 
-const PoiInformation = ({ dispatch, specificPoiInfos }) => (
-  <div className="informationPage">
+const PoiInformation = ({ dispatch, specificPoiInfos, InformationPoiInfos }) => (
+  <div>
     <button
-      className="closeButton"
+      className={InformationPoiInfos ? 'closeButtonInformationPageTop' : 'closeButtonInformationPageBottom'}
       type="button"
       onClick={() => dispatch({ type: 'CLOSE_POI_INFOS', specificPoiInfos: [] })}
     >
 X
     </button>
-    <h1>{specificPoiInfos.name}</h1>
-    <hr />
-    <p>Adresse</p>
-    <p>code postal ville</p>
-    <p>Distance</p>
-    <div>
-      <img src={specificPoiInfos.picture_url} alt="table ping pong" />
-    </div>
-    <div className="informationUser">
-      <p>{specificPoiInfos.author}</p>
-      <p>
+    {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+    <div
+      className={InformationPoiInfos ? 'informationPageTop' : 'informationPageBottom'}
+      onClick={() => dispatch({ type: 'TRANSITION_POI_INFOS', InformationPoiInfos: !InformationPoiInfos })}
+      role="button"
+      tabIndex="0"
+    >
+      <h1>{specificPoiInfos.name}</h1>
+      <hr />
+      <p>Adresse</p>
+      <p>code postal ville</p>
+      <p>Distance</p>
+      <div>
+        <img src={specificPoiInfos.picture_url} className={InformationPoiInfos ? 'informationPicture' : 'informationPicture-Bottom'} alt="table ping pong" />
```

Comment:
```alt``` is not a dynamic value, I don't think you will always have a picture of a ping pong table

Reply:
Modification ok